src/components/CommentAdder.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState, useContext } from "react";
+import { postNewComment } from "../utils/post";
+import UserContext from "../contexts/UserContext";
+import { Box } from "@mui/material";
+
+export default function CommentAdder({
+  articleId,
+  setCommentsList,
+  commentsList,
+}) {
+  const { username } = useContext(UserContext);
+  const [newCommentBody, setNewCommentBody] = useState("");
+  const [newCommentAuthor, setNewCommentAuthor] = useState(username);
+  const [newCommentVotes, setNewCommentVotes] = useState(0);
+  const [newCommentArticleId, setNewCommentArticleId] = useState(articleId);
+  const [err, setErr] = useState(null);
+  const [success, setSuccess] = useState(null);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    let newCommentObject = {};
+    newCommentObject.body = newCommentBody;
+    newCommentObject.username = newCommentAuthor;
+    setCommentsList([...commentsList], [newCommentObject]);
+    setErr(null);
+    {
+      newCommentObject.body.length < 10
+        ? setErr("Error: Please type in a comment of at least 10 characters")
+        : setSuccess("Your comment has been posted!");
+    }
+    if (err !== null) {
```

Comment:
should be === 
I have changed this now and it's fixed the bug of comments not appearing